<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClinicDoctor</class>
 <widget class="QMainWindow" name="ClinicDoctor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>953</width>
    <height>750</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Call A Doctor - Clinic Doctor Page</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="SpecialityList">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>290</y>
      <width>381</width>
      <height>131</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>General Practice</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Cardiology</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Dermatology</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Endocrinology</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Neurology</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Obstetrics and Gynecology</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Orthopedic Surgery</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Pediatrics</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Psychiatry</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Radiology</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Surgery (General)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Urology</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="SearchDoctorInput">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>291</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Search Doctor Name Here</string>
    </property>
    <property name="clearButtonEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="SortSpecialityLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>140</y>
      <width>271</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Sort By Speciality</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Search_pushButton">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>80</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Search</string>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="DashboardButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Dashboard</string>
    </property>
   </widget>
   <widget class="QPushButton" name="DoctorsButton">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>10</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Doctors</string>
    </property>
   </widget>
   <widget class="QPushButton" name="RequestsButton">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>10</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Requests</string>
    </property>
   </widget>
   <widget class="QPushButton" name="LogoutButton">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>10</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Logout</string>
    </property>
   </widget>
   <widget class="QComboBox" name="Specility">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>170</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QTreeWidget" name="DoctorTree">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>230</y>
      <width>471</width>
      <height>331</height>
     </rect>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
    <column>
     <property name="text">
      <string>Doctor Name</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Status</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Speciality</string>
     </property>
    </column>
    <item>
     <property name="text">
      <string/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="text">
      <string/>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="ToggleStatusButton">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>580</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Toggle Status</string>
    </property>
   </widget>
   <widget class="QPushButton" name="RegisterDoctorButton_2">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>670</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Register Doctor</string>
    </property>
   </widget>
   <widget class="QPushButton" name="DeleteDoctorButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>580</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Delete Doctor </string>
    </property>
   </widget>
   <widget class="QLabel" name="DoctorNameLabel">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>90</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Doctor Name:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="DoctorNameInput">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>130</y>
      <width>381</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Enter Doctor Name Here</string>
    </property>
   </widget>
   <widget class="QLabel" name="DoctorJobLabel">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>170</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Doctor Job:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="DoctorJobInput">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>210</y>
      <width>381</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Enter Doctor Job Here</string>
    </property>
   </widget>
   <widget class="QLabel" name="ChooseSpecialityLabel">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>250</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Choose Speciality:</string>
    </property>
   </widget>
   <widget class="QLabel" name="ManageDoctorslabel">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>50</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Manage Doctors</string>
    </property>
   </widget>
   <widget class="QLabel" name="DoctorEmailLabel">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>480</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Doctor Email:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="DoctorEmailInput">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>520</y>
      <width>381</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Enter Doctor Email Here</string>
    </property>
   </widget>
   <widget class="QLabel" name="DoctorPasswordLabel">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>560</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Doctor Password:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="DoctorPasswordInput">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>600</y>
      <width>381</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Enter Doctor Password Here</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ShowAllButton">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>170</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Show All</string>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="DoctorStatusLabel">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>430</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Doctor Status:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="DoctorStatuscomboBox">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>440</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <item>
     <property name="text">
      <string>Available</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Busy</string>
     </property>
    </item>
   </widget>
   <widget class="QCheckBox" name="ShowPassword">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>640</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Show Password</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
